src/Npgsql/TypeMapping/GlobalTypeMapper.cs

```diff
@@ -714,6 +713,14 @@ void SetupMiscHandlers()
                     NpgsqlDbType = NpgsqlDbType.Bit,
                     TypeHandlerFactory = new DefaultTypeHandlerFactory<BitStringHandler, BitArray>()
                 }.Build());
+                
+                this.AddMapping(new NpgsqlTypeMappingBuilder
```

Comment:
Should help with the test failures:

```suggestion
                AddMapping(new NpgsqlTypeMappingBuilder
```